api/src/main/java/io/strimzi/api/kafka/model/Zookeeper.java

```diff
@@ -39,7 +39,7 @@
 
     private Logging logging;
 
-    private String tlsSidecarImage;
+    private Sidecar tlsSidecar;
```

Comment:
Strictly this is a change in schema which would require a change in apiVersion. Since CRDs are new and we've not done a release I din't think we should worry about it this time. But we do need to be very careful when adding new properties (particularly scalar properties) that we're not going to want to change them later (to object properties). 

So really this is something the whole team should take much more care about.